bsp_diff/caas/device/intel/civ/host/vm-manager/0024-Script-changes-for-wifi-and-BT-firmware-update.patch

```diff
@@ -0,0 +1,31 @@
+From 4a1d22138852a0ff5799ebe98accf8c8a0973a99 Mon Sep 17 00:00:00 2001
+From: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
+Date: Thu, 6 Jan 2022 12:37:33 +0530
+Subject: [PATCH 2/3] Script changes for wifi and BT firmware update
+
+Tracked-On: OAM-100553
```

Comment:
OAM-100553 looks to be from Android 12.
Why do we need this particular firmware?
Why is it that this is not part of Android 13 horizontal and we need to port here?
Is there a Jira ticket submitted for horizontal?